test/units/cli/test_adhoc.py

```diff
@@ -93,7 +93,34 @@ def test_run_no_extra_vars():
     assert exec_info.value.code == 2
 
 
-def test_ansible_version(capsys, mocker):
+def test_ansible_git_version(capsys, mocker):
+    mocker.patch("ansible.cli.arguments.option_helpers._git_repo_info",
+                 return_value="ansible 2.11.0.dev0 (devel 8a202cae3e) last updated 2021/01/11 10:24:38 (GMT +200)")
+    adhoc_cli = AdHocCLI(args=['/bin/ansible', '--version'])
+    with pytest.raises(SystemExit):
+        adhoc_cli.run()
+    version = capsys.readouterr()
+    try:
+        version_lines = version.out.splitlines()
+    except AttributeError:
+        # Python 2.6 does return a named tuple, so get the first item
+        version_lines = version[0].splitlines()
+
+    assert len(version_lines) == 9, 'Incorrect number of lines in "ansible --version" output'
+    assert re.match('ansible [0-9.a-z]+ .*$', version_lines[0]), 'Incorrect ansible version line in "ansible --version" output'
```

Comment:
It's usually better to use raw-strings for regexps:
```suggestion
    assert re.match(r'ansible [0-9.a-z]+ .*$', version_lines[0]), 'Incorrect ansible version line in "ansible --version" output'
```
(I'm pretty sure Python 3.6+ will emit warnings if you don't)

Reply:
I would like to change all the regex strings to raw string then, any objections?

Reply:
will leave it non raw and change all of that regex lines in a later commit